wikipron/extract/khm.py

```diff
@@ -0,0 +1,14 @@
+"""Word and pron extraction for Khmer."""
+
+import itertools
+
+from wikipron.extract.default import yield_pron
+
+
+_IPA_XPATH = '//span[@class = "IPA" and @lang = "km"]'
```

Comment:
This is pseudo-private because it's just for Khmer, is that right?

Could the khm module JUST contain the `IPA_XPATH` spec? OR could we just have `wikipron/extract/ipa_xpath.py` which containts `IPA_XPATH_DEFAULT`, `IPA_XPATH_KHMER`, and then momve `extract_word_pron` to `core.py`?